src/authorizeStep.js

```diff
@@ -18,8 +18,8 @@ export class AuthorizeStep {
         return next.cancel(new Redirect(loginRoute));
       }
     }
-    else if (isLoggedIn && routingContext.getAllInstructions().some(i => i.fragment) == loginRoute) {
-      var loginRedirect = this.auth.getLoginRedirect();
+    else if (isLoggedIn && routingContext.getAllInstructions().some(i => i.fragment == loginRoute)) {
+      var loginRedirect = this.auth.getLoginRedirect().replace(/\/#/gi, '');
```

Comment:
I don't think this belongs here, personally. I think this should be changed in the configuration in stead of using a hack here.